### ISSUE TITLE
Resolution disappears from Action when restarting LTT or Windows

### ISSUE THREAD

**Issue by AlaDragon** [BartoszCichecki/LenovoLegionToolkit#907]
### Rules

- [X] I made myself familiar with the README.
- [X] I checked closed issues for similar ones and did not find a similar problem mentioned before.
- [X] I understand that, if insufficient information was provided, my issue will be closed without an answer.

### Version

2.15.4

### OS

Windows 10 22H2 Build 19045.3208

### Device

Legion 5 15ITH6H

### BIOS version

H1CN53WW

### What's wrong?

Recently whenever I set an action that changes display resolution and I restart LTT or the laptop, that action would lose the display resolution choice and become blank and I have to manually re-add the resolution to the action every time.
Also, I verified and it's not a visual bug, the action executes without changing resolution if it does not appear in the action.

Here's how it appears before restarting:
![image](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/118928622/0cce818e-8764-4c5e-b0b7-97440a2861c2)

And here's after I restart:
![image](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/118928622/a41b62a4-4e13-45ca-81ac-ad2e8dd70bf6)

### How to reproduce the bug?

1. Add Resolution to an Action
2. Save
3. Restart LLT or Laptop
4. Resolution disappears from the Action

### What is the behavior that you expected?

Resolution should stay in the action like before.

### Logs

First log is when I launch LTT and set the Action with Resolution and then I quit LTT: log_2023_08_01_17_18_40.txt
Second log is when I re-launch LTT to verify if the Action still have Resolution or not: log_2023_08_01_17_19_43.txt
[log_2023_08_01_17_18_40.txt](https://github.com/BartoszCichecki/LenovoLegionToolkit/files/12232421/log_2023_08_01_17_18_40.txt)
[log_2023_08_01_17_19_43.txt](https://github.com/BartoszCichecki/LenovoLegionToolkit/files/12232425/log_2023_08_01_17_19_43.txt)

### Additional information

No idea why it stopped working, I remember the action to auto control resolution working fine weeks ago, I also haven't done any windows or Nvidia update since June.

**Comment by AlaDragon** [comment]
I also checked the automation.json and the action in screenshot has the following steps:
```
      "Steps": [
        {
          "$type": "LenovoLegionToolkit.Lib.Automation.Steps.PowerModeAutomationStep, LenovoLegionToolkit.Lib.Automation",
          "State": "Quiet"
        },
        {
          "$type": "LenovoLegionToolkit.Lib.Automation.Steps.ResolutionAutomationStep, LenovoLegionToolkit.Lib.Automation",
          "State": {}
        },
        {
          "$type": "LenovoLegionToolkit.Lib.Automation.Steps.DpiScaleAutomationStep, LenovoLegionToolkit.Lib.Automation",
          "State": {
            "Scale": 125
          }
        }
      ],
```
I noticed that the State for the resolution Step is empty, I thought maybe I can add a resolution there manually and make the file read-only as a work around, but I don't know what exactly to write there.

**Comment by BartoszCichecki** [comment]
Hey, thanks for reporting. This issue was already fixed in db196b9e44e38af7fccc05e9ebb6d32a99ac4dc7 and will be released in 2.16.0. Beta version is already posted on Discord linked in the readme in case you want to try it out.
